dco_check/dco_check.py

```diff
@@ -931,6 +931,16 @@ def get_commit_range(self) -> Optional[Tuple[str, str]]:  # noqa: D102
                 f"\ton pull request branch '{commit_branch_head}': "
                 f"will check commits off of base branch '{commit_branch_base}'"
             )
+        elif event_name in 'merge_group':
+            # See: https://docs.github.com/en/webhooks/webhook-events-and-payloads#merge_group
+            commit_hash_base = self.event_payload['merge_group']['base_sha']
+            commit_hash_head = self.event_payload['merge_group']['head_sha']
+            commit_branch_base = self.event_payload['merge_group']['base_ref']
+            commit_branch_head = self.event_payload['merge_group']['head_ref']
+            logger.verbose_print(
+                f"\ton merge request branch '{commit_branch_head}': "
```

Comment:
Do you think it makes sense to make this message a bit different than the one for `event_name in ('pull_request', 'pull_request_target')`, this way we can know if it detected the `merge_group` correctly?